content/js3/blocks/using-fetch/index.md

```diff
@@ -0,0 +1,53 @@
++++
+title = '🌐 Requesting from a server side API'
+headless = true
+time = 20
+facilitation = false
+emoji= '🧩'
+[objectives]
+1="List 4 preceding concepts of asynchronous programming in JavaScript"
```

Comment:
Which concepts are these? I'm not sure what they are from the current contents of the prep page...